ui/app/templates/vault/cluster.hbs

```diff
@@ -85,15 +85,26 @@
       {{#if flash.componentName}}
         {{component flash.componentName content=flash.content}}
       {{else}}
-        <h5 class="title is-5 has-text-{{if (eq flash.type 'warning') 'dark-yellow' flash.type}}">
-          {{get (message-types flash.type) "text"}}
-        </h5>
-        <span data-test-flash-message-body=true>
```

Comment:
Oh ha, didn't catch this one - we use that data-test selector for a lot of the acceptance tests.

Reply:
Oh, oops. I had been trying to use the <AlertBanner> component here and it's already in there, but I somehow missed including it again after I switched back. Got it fixed in the empty states branch